packages/plugin-ext/src/hosted/node/scanners/scanner-theia.ts

```diff
@@ -121,9 +121,9 @@ export class TheiaPluginScanner implements PluginScanner {
         }
 
         const contributions: PluginContribution = {};
-        if (rawPlugin.contributes.configuration) {
-            const config = this.readConfiguration(rawPlugin.contributes.configuration, rawPlugin.packagePath);
-            contributions.configuration = config;
+        const config = rawPlugin.contributes.configuration;
+        if (config) {
+            contributions.configuration = this.readConfiguration(Array.isArray(config) ? config[0] : config, rawPlugin.packagePath);
```

Comment:
It's better to merge the config array if it contains more than one element.
I will update this PR.